governance/program/src/state/enums.rs

```diff
@@ -15,8 +15,8 @@ pub enum GovernanceAccountType {
     /// Voter record for each voter and given governing token type within a Realm
     VoterRecord,
 
-    /// Program Governance account
-    ProgramGovernance,
+    /// Account Governance account
+    AccountGovernance,
```

Comment:
Since program governances and account governances are not the same thing, mainly because they're derived using different seeds, we should have at least two different enums.  I think that'll make frontend integration easier.  When querying, we'll know explicitly if it's a program or account governance.  We can probably reuse the same type.  I'm not fussed either way

Reply:
I've been thinking about this because I didn't like  `AccountGovernance` name which felt too specific to me. 
I renamed it to be more generic `Governance` and added `AccountGovernance` and `ProgramGovernance` as account types. This way we can have multiple types using the same struct.
With a single struct covering different types we can write generic code in other places where the type is not important, for example when creating `Proposals`.